README.md

```diff
@@ -78,9 +78,11 @@ Single Message Transforms, or SMTs, can be used to customize fields or values of
     transforms.ReplaceField.type=org.apache.kafka.connect.transforms.ReplaceField$Value 
     transforms.ReplaceField.exclude=_id
     ```
+1. If you have events where there is no value ([_tombstone_ events](https://kafka.apache.org/documentation.html#compaction)), you may wish to filter these out.
+    - In the Cloudant sink connector, these may be undesirable as they will generate an empty document.
+    - In the Cloudant source connector, tombstone events are generated by default for deleted documents (in addition to the deleted document itself).
```

Comment:
I think we should remove "by default" as we aren't providing any option to not generate them.
```suggestion
    - In the Cloudant source connector, tombstone events are generated for deleted documents (in addition to the deleted document itself).
```